source/connect/mongoclient.txt

```diff
@@ -18,54 +18,70 @@ Create a MongoClient
    :depth: 2
    :class: singlecol
 
-This guide shows you how to connect to a
-`MongoDB Atlas deployment <https://www.mongodb.com/docs/atlas>`__,
-a MongoDB instance, or a replica set using the Node.js driver.
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the
+  {+driver-short+} which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and performs 
```

Comment:
S: you can [remove the articles](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items) at the beginning of each bullet point
```suggestion
- **Connection URI**, also known as a *connection string*, which tells the
  {+driver-short+} which MongoDB deployment to connect to.
- **MongoClient** object, which creates the connection to and performs 
```